### ISSUE TITLE
about the results on semantic segmentation.

### ISSUE THREAD

**Issue by shuluoshu** [charlesq34/pointnet#63]
Hi, @charlesq34 , 
For the semantic segmentation task, the reported overall accuracy and IoU is 78.6% and 47.71% on S3DIS dataset. I wonder how you compute this in detail. Do you independently trained 6 models (for example, for testing area3, train on area 1,2,4,5,6 data) and perform testing on the untrained area. Then average the accuracy and IoU over the 6 areas ? Since I only test on the Area6 and got a higher accuracy and IoU than the given result. 
Besides, could you offer your pre-trained models of semantic segmentation task ? Since it is too long to train 6 models. 
 Thanks~

**Comment by jiayuanyuan1011** [comment]
Hello, @charlesq34 ,
I have a questions.When experiment in segmentation, Data preprocessing format is XYZRGBX'Y'Z', I want to know why we need the X'Y'Z', what's meaning and benefits of X'Y'Z'.

**Comment by charlesq34** [comment]
@jiayuanyuan1011 I've replied to the question in another thread. 

@shuluoshu sorry for the long delay in response. haven't checked the repo for a while. Your understanding is right. We currently don't have plans to release all these models.. Note that for the original dataset paper they have another way to split train/test set.

I'm closing the questions now. I can reopen it if necessary.

**Comment by Saira05** [comment]
i trained 6 models with 6 fold cross validation like test area=6 and train= 1,2,3,4,5 and then so on i got following results.
 mean IOU             0.4164943362443013
Overall accuracy: 0.749509875125143

I want to know i thing how can i use these models as feature extractor? is there any way?

**Comment by Q-Mao** [comment]
You can delete the last classify layer and then the output of the network will be the features

**Comment by ZoeZhang91** [comment]
I also trained and tested on the 6 models, finally the results turn out to be like the following：
       Overall accuracy : 0.8821330373219757
       mean IOU: 0.6663725902575812

**Comment by renmaqilong** [comment]
You should check your result carefully. It may be not so right.@ZoeZhang91
